tss-esapi/src/handles/tpm.rs

```diff
@@ -114,7 +114,7 @@ macro_rules! create_tpm_handle_type {
             pub fn new(value: u32) -> Result<$handle_type_name> {
                 if value.to_be_bytes()[0] != $tpm_handle_type_id {
                     error!(
-                        "Errro: TPM Handle ID of the input value did not match the {} (!={})",
+                        "Error: TPM Handle ID of the input value did not match the {} (!={})",
                         stringify!($handle_type_name),
```

Comment:
I know it is not really part of the ticket. But there is no need to write Error in a lock::error it will just result in something looking like ```rust-esapi-tss ERROR - Error ...." ``` so remove the error completely in the message.

Reply:
Yep. Good point :+1:

Reply:
There are quite a few of these in the codebase. I'll fix them all at once.